manila/share/drivers/vastdata/__init__.py

```diff
@@ -221,14 +242,13 @@ def ensure_share(self, context, share, share_server=None):
         elif not export.policy == share_id:
             self.vms_session.patch("views/{}".format(export.id), data=dict(policy_id=policy.id))
 
-        return [dict(
+        return dict(
```

Comment:
why? it used to return a `List[Dict]` type, how can we change it to a `Dict` now? Am I missing something?

(`ensure_share` isn't only used by `create_share`)

Reply:
Yes, it is strange for me. List of dict was generaged before only for reason to take first dict on line 250. And it aways returned first pool before with the same ip.

Reply:
if you `git grep` the manila project you'll find that the `ensure_share` is called from another location, where a list is expected. 
this is not the only place it is used.